index.js

```diff
@@ -0,0 +1,17 @@
+const sliderBlur = document.getElementById("sliderblur");
```

Comment:
Maybe change the variable names to things `sliderBlurInput` to make clear that they are elements?